test/test_validation.jl

```diff
@@ -18,4 +18,33 @@ using Test, Braket, BraketSimulator
q[1];\ncnot q[1], q[7];\nh q[7];\ncnot q[7], q[2];\nry(p_98) q[2];\nry(p_99) q[1];\ncnot q[6], q[2];\ncnot q[6], q[1];\nry(p_100) q[2];\nry(p_101) q[1];\ncnot q[7], q[2];\nh q[7];\ncnot q[1], q[7];\nry(p_102) q[2];\nry(p_103) q[1];\ncnot q[1], q[2];\ncnot q[6], q[1];\nh q[1];\nh q[7];\ncnot q[1], q[6];\ncnot q[6], q[7];\ncnot q[5], q[7];\ncnot q[1], q[5];\nh q[7];\nh q[1];\ncnot q[5], q[7];\ncnot q[1], q[3];\nry(p_104) q[3];\nry(p_105) q[1];\ncnot q[1], q[7];\nh q[7];\ncnot q[7], q[3];\nry(p_106) q[3];\nry(p_107) q[1];\ncnot q[5], q[3];\ncnot q[5], q[1];\nry(p_108) q[3];\nry(p_109) q[1];\ncnot q[7], q[3];\nh q[7];\ncnot q[1], q[7];\nry(p_110) q[3];\nry(p_111) q[1];\ncnot q[1], q[3];\ncnot q[5], q[1];\nh q[1];\nh q[7];\ncnot q[1], q[5];\ncnot q[5], q[7];\ncnot q[4], q[5];\ncnot q[2], q[4];\nh q[5];\nh q[2];\ncnot q[4], q[5];\ncnot q[2], q[3];\nry(p_112) q[3];\nry(p_113) q[2];\ncnot q[2], q[5];\nh q[5];\ncnot q[5], q[3];\nry(p_114) q[3];\nry(p_115) q[2];\ncnot q[4], q[3];\ncnot q[4], q[2];\nry(p_116) q[3];\nry(p_117) q[2];\ncnot q[5], q[3];\nh q[5];\ncnot q[2], q[5];\nry(p_118) q[3];\nry(p_119) q[2];\ncnot q[2], q[3];\ncnot q[4], q[2];\nh q[2];\nh q[5];\ncnot q[2], q[4];\ncnot q[4], q[5];\ncnot q[4], q[7];\ncnot q[2], q[4];\nh q[7];\nh q[2];\ncnot q[4], q[7];\ncnot q[2], q[3];\nry(p_120) q[3];\nry(p_121) q[2];\ncnot q[2], q[7];\nh q[7];\ncnot q[7], q[3];\nry(p_122) q[3];\nry(p_123) q[2];\ncnot q[4], q[3];\ncnot q[4], q[2];\nry(p_124) q[3];\nry(p_125) q[2];\ncnot q[7], q[3];\nh q[7];\ncnot q[2], q[7];\nry(p_126) q[3];\nry(p_127) q[2];\ncnot q[2], q[3];\ncnot q[4], q[2];\nh q[2];\nh q[7];\ncnot q[2], q[4];\ncnot q[4], q[7];\ncnot q[5], q[6];\ncnot q[2], q[5];\nh q[6];\nh q[2];\ncnot q[5], q[6];\ncnot q[2], q[3];\nry(p_128) q[3];\nry(p_129) q[2];\ncnot q[2], q[6];\nh q[6];\ncnot q[6], q[3];\nry(p_130) q[3];\nry(p_131) q[2];\ncnot q[5], q[3];\ncnot q[5], q[2];\nry(p_132) q[3];\nry(p_133) q[2];\ncnot q[6], q[3];\nh q[6];\ncnot q[2], q[6];\nry(p_134) q[3];\nry(p_135) q[2];\ncnot q[2], q[3];\ncnot q[5], q[2];\nh q[2];\nh q[6];\ncnot q[2], q[5];\ncnot q[5], q[6];\ncnot q[6], q[7];\ncnot q[2], q[6];\nh q[7];\nh q[2];\ncnot q[6], q[7];\ncnot q[2], q[3];\nry(p_136) q[3];\nry(p_137) q[2];\ncnot q[2], q[7];\nh q[7];\ncnot q[7], q[3];\nry(p_138) q[3];\nry(p_139) q[2];\ncnot q[6], q[3];\ncnot q[6], q[2];\nry(p_140) q[3];\nry(p_141) q[2];\ncnot q[7], q[3];\nh q[7];\ncnot q[2], q[7];\nry(p_142) q[3];\nry(p_143) q[2];\ncnot q[2], q[3];\ncnot q[6], q[2];\nh q[2];\nh q[7];\ncnot q[2], q[6];\ncnot q[6], q[7];\nry(p_144) q[3];\nry(p_145) q[7];\n#pragma braket result expectation z(q[3]) @ z(q[5]) @ z(q[6]) @ z(q[7])\n#pragma braket result expectation z(q[1]) @ z(q[3]) @ z(q[4]) @ z(q[5]) @ z(q[6]) @ z(q[7])\n#pragma braket result expectation z(q[2]) @ z(q[3]) @ z(q[4]) @ z(q[5]) @ z(q[6]) @ z(q[7])\n#pragma braket result expectation z(q[3]) @ z(q[4]) @ z(q[5]) @ z(q[6]) @ z(q[7])\n#pragma braket result expectation z(q[0]) @ z(q[3]) @ z(q[4]) @ z(q[5]) @ z(q[6]) @ z(q[7])
```

Comment:
Is there any way to make this more readable?

Reply:
Probably, let me do that now

Reply:
Done